specification/containerstorage/ContainerStorage.Management/main.tsp

```diff
@@ -22,6 +22,7 @@ namespace Microsoft.ContainerStorage;
 enum Versions {
   /** 2023-07-01-preview preview version */
   @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v4)
```

Comment:
This will change *all* the common-types to v4, not just the managedserviceidentity